scripts/install_libvirt_deps.sh

```diff
@@ -1,34 +1,7 @@
 #!/bin/bash
 
-if [ $# -lt 1 ]; then
-  echo "Usage $0 <lp-path>"
-  echo
-  echo lp-path: path to linchpin source
-  exit 1
-fi
+dnf install libvirt-devel -yq
+dnf install libguestfs-tools python-libguestfs
+pip install linchpin[libvirt]
 
-LP_PATH=${1}
 
-VENV_LIB_PATH=lib/python2.7/site-packages
-LIBSELINUX_PATH=/usr/lib64/python2.7/site-packages
-
-if [ -n "${VIRTUAL_ENV}" ]; then
-
-    if [ "${USER}" != "root" ]; then
-        sudo dnf install libvirt-devel libselinux-python -yq &> /dev/null
-    else
-        dnf install libvirt-devel libselinux-python -yq &> /dev/null
-    fi
-
-    ln -s ${LIBSELINUX_PATH}/selinux ${VIRTUAL_ENV}/${VENV_LIB_PATH} &> /dev/null
```

Comment:
It makes sense to have the libselinux stuff separate for other items. But since it is also a prerequisite for libvirt to work, why not have this script now call the one you moved out of this one?